### PR TITLE
Split docs homepage into separate pages in a 'Getting started' section

### PR DESCRIPTION
**Pull request by melinath**
I split the docs homepage into separate pages in a 'Getting started' section. I lightly edited the content to account for the changes, since people will be access it repeatedly rather than as a one-time guide. I've omitted the following sections:

- Submitting a PR (duplicates "contributing" and is outdated)
- Detailed contributing steps (duplicates sidebar)
- Glossary (outdated)

```release-note:none

```

### CODE REVIEW

**Comment by ScottSuarez** [comment]
Is there a way to spin this up locally so I can view the generated changes locally?

**Comment by melinath** [comment]
@ScottSuarez Install Hugo and then run `hugo serve` from the docs directory.

**Review by ScottSuarez — APPROVED** [review]
overall things look good to me. I haven't reviewed the content of the articles as there is a lot and it's not clear to me what has changed. If you would like me to pass over the content let me know. 

I would recommend consistency in the index pages. If we want navigation headers to give summaries of the nested content (which I like). I would prefer to consistently do that for each header. This is a stylistic preference and a non-blocker.

File: docs/content/docs/getting-started/_index.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
```
[the remark this answers is not in the document]
Comment (melinath) [reply docs/content/docs/getting-started/_index.md:1]: I'm not sure yet if it'll make sense to have all of the sections have summaries like this or if "how to" is an exception... I'd like to leave it as-is for now.

**Pull request merged**